airbyte-cdk/python/airbyte_cdk/sources/utils/schema_helpers.py

```diff
@@ -157,7 +158,11 @@ def check_config_against_spec_or_exit(config: Mapping[str, Any], spec: Connector
     try:
         validate(instance=config, schema=spec_schema)
     except ValidationError as validation_error:
-        raise Exception("Config validation error: " + validation_error.message) from None
+        raise AirbyteTracedException(
+            message="Config validation error: " + validation_error.message,
+            internal_message=validation_error.message,
+            failure_type=FailureType.config_error,
+        ) from None
```

Comment:
We're still using `from None` here to avoid logging out secrets since the stack trace for the ValidationError contains the values from the provided config

Reply:
can you add that comment in the code?